template/data/fr/global.json

```
@@ -0,0 +1,116 @@
+{
+  "globalFR": {
+    "id": 2,
+    "locale": "fr",
+    "metaTitleSuffix": "Strapi Corporate",
+    "metadata": {
+      "metaTitle": "Strapi starter for Corporate Sites",
+      "metaDescription": "Créer votre site corporate avec Strapi",
+      "twitterCardType": "summary",
+      "twitterUsername": "strapijs",
+      "shareImage": null
+    },
+    "notificationBanner": {
+      "text": "Cette page a été crée avec le Strapi starter pour les sites Corporate.\n [Voir les autres Strapi starters](https://strapi.io/starters)",
```

I think it sounds more natural like this in french:

```suggestion
      "text": "Cette page a été crée avec le Strapi starter pour les sites Corporate.\n [Voir les autres starters Strapi](https://strapi.io/starters)",
```